compiler/tflchef/tflite/src/Op/Abs.cpp

```diff
@@ -28,9 +28,18 @@ void TFliteOpAbs::filler(const tflite::Operator *op, TFliteImport *import,
 }
 
 tflchef::Operation *TFliteOpAbs::build(const tflite::Operator *op, TFliteImport *import,
-                                       tflchef::ModelRecipe *model_recipe) const
+                                       tflchef::ModelRecipe *model_recipe,
+                                       tflchef::Graph *model_graph) const
 {
-  auto operation = model_recipe->add_operation();
+  tflchef::Operation *operation = nullptr;
+  if (model_graph)
+  {
+    operation = model_graph->add_operation();
+  }
+  else
+  {
+    operation = model_recipe->add_operation();
+  }
```

Comment:
For adding operation to subgraph or to main graph

Reply:
Is not not possible to extend `TFliteImport` ?
I don't want all these source changed by changing `build()` prototype.

Reply:
I see that `TFliteImport` is for tflite model as a reader and `tflchef::ModelRecipe` and `tflchef::Graph` are where new op should be created.
These two exist in different context, so using `TFliteImport` doesn't seem to be nice.

Current `if (model_graph)` check is simple and direct but I didn't expect like this.

I need to think about this...